sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/BeamSqlDslSqlStdOperatorsTest.java

```diff
@@ -1364,6 +1364,18 @@ public void testTimestampMinusInterval() {
     checker.buildRunAndCheck();
   }
 
+  @Test
+  // More needed @SqlOperatorTest(name = CONVERT_TIMEZONE, kind = "DATETIME")
+  public void testConvertTimezone() {
+    ExpressionChecker checker =
+        new ExpressionChecker()
+            .addExpr(
+                "CONVERT_TIMEZONE('America/New York', 'UTC', TIMESTAMP '1984-04-19 01:01:58')",
+                parseTimestampWithUTCTimeZone("1984-04-19 01:01:56"));
```

Comment:
Noting that I wasn't trying to actually get the answer right - I just wanted to get it to parse, then sort out the right answer. I find timestamp instantiation while mystifying in Calcite.